src/cformat.jl

```diff
@@ -1,16 +1,80 @@
 formatters = Dict{ Compat.ASCIIString, Function }()
 
-function sprintf1( fmt::Compat.ASCIIString, x )
+@static if VERSION >= v"0.6.0-dev.1671"
```

Comment:
I'd believed it was the preferred way to deal with sections of code that should not be compiled because of platform or version differences.
Is that not the case?